library/src/main/java/com/yuyh/library/EasyGuide.java

```diff
@@ -1,384 +1,416 @@
-
-        mParentView = (FrameLayout) mActivity.getWindow().getDecorView();
-
+        this.dismissAnyWhere = dismissandperformclick[0];
+        this.performViewClick = dismissandperformclick[1];
+        mParentView = componentContainer;
     }
 
+
     /**
-     * 设置引导提示 状态改变(显示/取消) 监听
+     * Add the listener
      *
-     * @param listener
+     * @param listener listener
      */
     public void setOnStateChangedListener(OnStateChangedListener listener) {
         this.listener = listener;
     }
 
     /**
-     * 显示引导提示
+     * Show dialog.
      */
     public void show() {
-        mGuideView = new EasyGuideView(mActivity);
+        mGuideView = new EasyGuideView(mAbilitySlice);
         mGuideView.setHightLightAreas(mAreas);
-
-        mTipView = new LinearLayout(mActivity);
-        mTipView.setGravity(Gravity.CENTER_HORIZONTAL);
-        mTipView.setLayoutParams(new RelativeLayout.LayoutParams(MATCH_PARENT, WRAP_CONTENT));
-        mTipView.setOrientation(LinearLayout.VERTICAL);
-
+        DirectionalLayout tipView = new DirectionalLayout(mAbilitySlice);
+        tipView.setAlignment(LayoutAlignment.HORIZONTAL_CENTER);
+        tipView.setLayoutConfig(new DirectionalLayout.LayoutConfig(
+                ComponentContainer.LayoutConfig.MATCH_PARENT, ComponentContainer.LayoutConfig.MATCH_CONTENT));
+        tipView.setOrientation(DirectionalLayout.VERTICAL);
         if (mIndicators != null) {
             for (TipsView tipsView : mIndicators) {
-                addView(tipsView.view, tipsView.offsetX, tipsView.offsetY, tipsView.params);
+                addView(tipsView.getView(), tipsView.getOffsetX(), tipsView.getOffsetY(), tipsView.getParams());
             }
         }
-
         if (mMessages != null) {
-            int padding = dip2px(mActivity, 5);
+            int padding = dip2px(mAbilitySlice, 5);
             for (Message message : mMessages) {
-                TextView tvMsg = new TextView(mActivity);
-                tvMsg.setLayoutParams(new ViewGroup.LayoutParams(MATCH_PARENT, WRAP_CONTENT));
+                Text tvMsg = new Text(mAbilitySlice);
+                tvMsg.setLayoutConfig(new ComponentContainer.LayoutConfig(
+                        ComponentContainer.LayoutConfig.MATCH_PARENT, ComponentContainer.LayoutConfig.MATCH_CONTENT));
                 tvMsg.setPadding(padding, padding, padding, padding);
-                tvMsg.setGravity(Gravity.CENTER);
-                tvMsg.setText(message.message);
-                tvMsg.setTextColor(Color.WHITE);
-                tvMsg.setTextSize(message.textSize == -1 ? 12 : message.textSize);
-
-                mTipView.addView(tvMsg);
+                tvMsg.setTextAlignment(TextAlignment.CENTER);
+                tvMsg.setText(message.getMessagetext());
+                Color hmosColor = EasyGuide.changeParamToColor(ohos.agp.utils.Color.WHITE.getValue());
+                tvMsg.setTextColor(hmosColor);
+                tvMsg.setTextSize(50);
```

Comment:
textSize should not hardcoded. It should depend on message.textSize value. Please check the base lib implementation